src/vconsole/vconsole-setup.c

```diff
@@ -280,22 +275,22 @@ static void setup_remaining_vcs(int fd, bool utf8) {
         if (cfo.op != KD_FONT_OP_SET)
                 log_warning("Fonts will not be copied to remaining consoles");
 
-        for (i = 1; i <= 63; i++) {
-                char ttyname[strlen("/dev/tty") + DECIMAL_STR_MAX(int)];
+        for (unsigned i = 1; i <= 63; i++) {
```

Comment:
Unfortunately Lennart does not like C99, please declare the variable on top of the block.